src/qt_gui/settings_dialog.cpp

```diff
@@ -240,14 +240,14 @@ SettingsDialog::SettingsDialog(std::span<const QString> physical_devices,
             QString userDir;
             Common::FS::PathToQString(userDir, std::filesystem::current_path() / "user");
             if (std::filesystem::exists(std::filesystem::current_path() / "user")) {
-                QMessageBox::information(NULL, "Cannot create portable user folder",
-                                         userDir + " already exists");
+                QMessageBox::information(NULL, tr("Cannot create portable user folder"),
+                                         userDir + " " + tr("already exists"));
```

Comment:
`userDir` should be passed as an argument.

Reply:
Yes, you must use arguments like this for the translation to be consistent in all languages.
```cpp
                QMessageBox::information(
                    NULL, tr("Cannot create portable user folder %1 already exists"), userDir);
```